lib/gooddata/bricks/middleware/logger_middleware.rb

```diff
@@ -18,7 +18,7 @@ def call(params)
           logger = NilLogger.new
         else
           logger = params[:GDC_LOGGER_FILE].nil? ? Logger.new(STDOUT) : Logger.new(params[:GDC_LOGGER_FILE])
-          logger.level = params['GDC_LOG_LEVEL'] if params['GDC_LOG_LEVEL']
+          logger.level = params['GDC_LOG_LEVEL'] || 'info'
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)